pkg/registrationserver/registrationserver.go

```diff
@@ -0,0 +1,126 @@
+package registrationserver
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	"github.com/gorilla/handlers"
```

Comment:
This needs to be "handlers", not "Handlers". It produces weird issues, looks like they have some case sensitivity in there.